backends/multidimensional_graphite.js

```diff
@@ -1,10 +1,47 @@
-var util = require('util');
-var events = require('events');
+var util = require('util')
+  , events = require('events')
+  , logger = require('../lib/logger')
+  , getHostname = require('os').hostname
+  , getBasename = require('path').basename;
 
 var log = console; // the logger
 var debug;
 
 
+function  createBackendLogger(config, logger) {
```

Comment:
also isn't this taken care of in the modification that is in the stats.js? It creates a logger with this ability for each of the backends